### PR TITLE
feat(tokens): add new tokens

### PR DESCRIPTION
**Pull request by rzpcibot**
This PR was opened by the Token Upload GitHub action. It updates source token files based on the payload from Figma Plugin.

### CODE REVIEW

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Review by kamleshchandnani — APPROVED** [review]

File: packages/blade/src/tokens/theme/paymentTheme.ts [hunk 1]
```diff
@@ -1833,19 +1781,19 @@ const colors: ColorsWithModes = {
             primary: {
               default: {
                 lowContrast: globalColors.chromatic.sapphire[500],
-                highContrast: globalColors.chromatic.sapphire[600],
+                highContrast: globalColors.chromatic.sapphire[50],
```
Comment (kamleshchandnani) [inline packages/blade/src/tokens/theme/paymentTheme.ts:1784]: why this change? this is being done by plugin already right?

Comment (chaitanyadeorukhkar) [reply packages/blade/src/tokens/theme/paymentTheme.ts:1784]: Plugin has only done this not me 🤔

**Review by kamleshchandnani — APPROVED** [review]
